BuildResidentialScheduleFile/measure.rb

```diff
@@ -0,0 +1,182 @@
+# frozen_string_literal: true
+
+# see the URL below for information on how to write OpenStudio measures
+# http://nrel.github.io/OpenStudio-user-documentation/reference/measure_writing_guide/
+
+require 'openstudio'
+require 'oga'
+
+require_relative 'resources/schedules'
+
+require_relative '../HPXMLtoOpenStudio/resources/constants'
+require_relative '../HPXMLtoOpenStudio/resources/geometry'
+require_relative '../HPXMLtoOpenStudio/resources/hpxml'
+require_relative '../HPXMLtoOpenStudio/resources/lighting'
+require_relative '../HPXMLtoOpenStudio/resources/meta_measure'
+require_relative '../HPXMLtoOpenStudio/resources/schedules'
+require_relative '../HPXMLtoOpenStudio/resources/xmlhelper'
+
+# start the measure
+class BuildResidentialScheduleFile < OpenStudio::Measure::ModelMeasure
+  # human readable name
+  def name
+    return 'Schedule File Builder'
+  end
+
+  # human readable description
+  def description
+    return 'Builds a residential schedule file.'
+  end
+
+  # human readable description of modeling approach
+  def modeler_description
+    return ''
```

Comment:
Add some description about how it creates a CSV file and inserts the CSV schedule file path into the HPXML file.